CITATION.cff

```diff
@@ -0,0 +1,132 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: UltraScan-III
+message: >-
+  A comprehensive data analysis framework for analytical
+  ultracentrifugation experiments, small angle scattering
+  experiments, and hydrodynamic bead modeling from atomic
+  structures
+type: software
+authors:
+  - given-names: Borries
+    family-names: Demeler
+    name-particle: Borries
+    email: demeler@gmail.com
+    affiliation: University of Montana
+    orcid: 'https://orcid.org/0000-0002-2414-9518'
+  - given-names: Emre
+    family-names: Brookes
+    email: emre.brookes@umontana.edu
+    affiliation: University of Montana
+    orcid: 'https://orcid.org/0000-0001-7348-6476'
+  - given-names: Gary
+    name-particle: E
```

Comment:
the name-particle are according to the docs for stuff like for example `von` or `zu` (German Adelstitle). I think it belongs to the given-names here.
```suggestion

```